Mail/Utils/DraftUtils.swift

```diff
@@ -30,13 +30,8 @@ enum DraftUtils {
         composeMessageIntent: Binding<ComposeMessageIntent?>
     ) {
         guard let message = thread.messages.first else { return }
-        // If we already have the draft locally, present it directly
-        if let draft = mailboxManager.draft(messageUid: message.uid)?.detached() {
-            matomoOpenDraft(isLoadedRemotely: false)
-            composeMessageIntent.wrappedValue = ComposeMessageIntent.existing(draft: draft, originMailboxManager: mailboxManager)
-        } else {
-            DraftUtils.editDraft(from: message, mailboxManager: mailboxManager, composeMessageIntent: composeMessageIntent)
-        }
+
+        DraftUtils.editDraft(from: message, mailboxManager: mailboxManager, composeMessageIntent: composeMessageIntent)
```

Comment:
Revert

Reply:
Context: We had seen together (IRL) that there was a bug: When we modify a draft, it appeared twice (the modified version and the old one) (duplicate bug)
And when we reverted that part of the code it seemed to work again.
Update: I tried to reproduce the bug but I can't do it, it would be necessary to investigate a little more